webapp/pages/terms-and-conditions.vue

```diff
@@ -1,14 +1,72 @@
 <template>
   <div>
     <ds-space>
-      <ds-heading tag="h2">{{ $t('site.termsAc') }}</ds-heading>
+      <ds-heading tag="h2">{{ $t('site.termsAc') }} ( vom: {{ latestDate_TaC }})</ds-heading>
     </ds-space>
+    <div v-if="confirmed === true">
+      <ds-section class="hasConfirm" primary>
+        <ds-flex>
+          <ds-flex-item>
+            <ds-logo inverse></ds-logo>
+          </ds-flex-item>
+          <ds-flex-item width="4">
+            <ds-heading>you have confirmed</ds-heading>
+            <ds-text>
+              You have confirmed the current Terms and Conditions on
```

Comment:
```suggestion
  {{ $t('terms-and-conditions.confirmed') }}
```
You want the text to be translated. You can also pass variables to the call of `$t`. Check out our examples by looking into our code, from folder `webapp` run `git grep '$t(`'` this should find all the lines of code where we use translations, including where we parameterize the translations.